de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/session/NegotiationHandler.java

```diff
@@ -47,31 +49,33 @@ public void handleOutgoingSessionNegotiation(
             public void run() {
                 SessionNegotiation.Status status;
                 if (ServerConfig.isInteractive()) {
-                    status = negotiation
-                        .start(new ConsoleProgressIndicator(System.out));
+                    status = negotiation.start(new ConsoleProgressIndicator(
+                        System.out));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWcmJq5RkEeD8QRkTSvp&open=AWcmJq5RkEeD8QRkTSvp)

[ISSUE_KEY]: AWcmJq5RkEeD8QRkTSvp